lib/install/action/extract.js

```diff
@@ -28,7 +30,16 @@ function andUpdatePackageJson (pkg, buildpath, next) {
 }
 
 function andStageBundledChildren (pkg, buildpath, log, next) {
-  var staging = path.resolve(buildpath, '..')
+  // Dependencies bundled inside a scoped module need an extra step up when
+  // resolving the path to the staging directory.
+  var staging = buildpath
+  do {
+    staging = path.resolve(staging, '..')
+  } while (staging.length > 1 && path.basename(staging) !== '.staging')
+  if (staging.length <= 1) {
+    throw new Error('???')
```

Comment:
Iit looks like you want a fatal error, for that you'd call the `next` callback with an error object as its first argument.

Output is mediated via `lib/utils/error-message.js`